zql/docs/search-syntax/README.md

```diff
@@ -318,7 +315,45 @@ http  2018-03-24T17:17:47.561097Z CgbtuX3gXoYFmEF82l 10.164.94.120 37311     10.
 http  2018-03-24T17:17:47.57066Z  CgbtuX3gXoYFmEF82l 10.164.94.120 37311     10.47.3.142 8080      26          GET    10.47.3.142 /scripts/waE8_BuNCEKM.shtml -        1.1     Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0) -      0                1635              404         Not Found  -         -        (empty) -        -        -       -          -              -               FdKLBd3fhPSqFIDFWc -              text/html
 ```
 
-Determining whether the value of a Zeek `addr`-type field is within a subnet also uses the pattern matching operator.
+### Containment
+
+Rather than testing for strict equality or pattern matches, you may want to determine if a value is among the many possible elements of a complex field. This is performed with the `in` operator.
+
+Our Zeek `dns` events include the `answers` field, which is an array of the multiple responses that may have been returned for a query. To determine which responses included hostname `e5803.b.akamaiedge.net`, we'll use `in`.
+
+#### Example:
+```zq-command
+zq -f table '"e5803.b.akamaiedge.net" in answers' dns.log.gz
```

Comment:
Just a side comment, so no problem here, but the double-quotes in single quotes pattern looks bad. I don't have a better suggestion ATM.